.eslintrc.js

```diff
@@ -4,16 +4,31 @@ module.exports = {
     node: true
   },
   extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:vue/base',
     'plugin:vue/essential',
+    // 'plugin:vue/recommended',
+    // 'plugin:vue/strongly-recommended',
+    'plugin:vue/vue3-essential',
+    // 'plugin:vue/vue3-recommended',
+    // 'plugin:vue/vue3-strongly-recommended',
+    'plugin:vuetify/base',
+    'plugin:vuetify/recommended',
     '@vue/standard',
     '@vue/typescript'
   ],
   rules: {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'max-len': ['warn', { code: 120, ignoreRegExpLiterals: true }],
+    // 'vue/attribute-hyphenation': 'off',
     // 'vue/no-deprecated-filter': 'warn',
+    'vue/no-deprecated-slot-scope-attribute': 'warn',
     // 'vue/no-deprecated-v-on-native-modifier': 'warn',
+    'vue/no-v-for-template-key-on-child': 'warn',
+    'vue/no-v-html': 'off',
+    'vue/v-on-event-hyphenation': 'off',
```

Comment:
I don't want the eslint plugins to auto-fix `@myEvent=` to `@my-event=`.

(In a future commit, I disable the rule on line 25 too.)